### DISCUSSION TITLE
PVR list location

### DISCUSSION THREAD

**Discussion by AdamMannPro** [Ascoware/get-iplayer-automator#483] [Q&A]
Hi all, I need to reload my macOS and would like to backup my PVR list. I have checked the previous location I had noted down but there is no folder there "~/Library" but this "Get iPlayer Automator" is not there anymore.

Same would be for the "Downloads History"

I have an old list but it is about half of the PVR I currently have.

If anyone can help me locate the current PVR file it would be greatly appreciated

**Comment by skovatch** [comment]
It's `~/Library/Application Support/Get iPlayer Automator/Queue.automatorqueue`. That file contains your PVR list and saved queue. `download_history` in the same directory is the download history. 

You should just save the entire directory and restore it.
